components/carousel/utils/useCarousel.ts

```diff
@@ -140,7 +144,10 @@ export const useCarouselRelated = ({ collectionId }) => {
 
   watch(data, async () => {
     if (data.value) {
-      nfts.value = await formatNFT((data.value as Collections).collection?.nfts)
+      const collection = await formatNFT(
```

Comment:
I would replace "const collection" with something more meaningful as it does not contain the collection data but rather the nfts.

Reply:
Yep, you're right, it's not collection, it's list of relation NFTs for carousel